bats/schema-import.bats

```diff
@@ -3,14 +3,39 @@ load $BATS_TEST_DIRNAME/helper/common.bash
 
 setup() {
     setup_common
+
+    cat <<DELIM > 1pk5col-ints.csv
+pk,c1,c2,c3,c4,c5
+0,1,2,3,4,5
+1,1,2,3,4,5
+DELIM
+
+    cat <<DELIM > 1pksupportedtypes.csv
+pk, int, string, boolean, float, uint, uuid
+0, 0, "asdf", TRUE, 0.0, 0, "00000000-0000-0000-0000-000000000000"
+1, -1, "qwerty", FALSE, -1.0, 1, "00000000-0000-0000-0000-000000000001"
+2, 1, "", TRUE, 0.0, 0, "123e4567-e89b-12d3-a456-426655440000"
+DELIM
+
+    cat <<DELIM > abc.csv
+pk,a,b,c
+0, red,  1.1, true
+1, blue, 2.2, false
+DELIM
+
+    cat <<DELIM > abc-xyz.csv
+pk,a,b,c,x,y,z
+0, red,  1.1, true,  green,  3.14, -1
+1, blue, 2.2, false, yellow, 2.71, -2
+DELIM
 }
 
 teardown() {
```

Comment:
Should you blow away the files in teardown?

Reply:
the whole directory for each test gets deleted when it completes